application/src/main/java/org/opentripplanner/transit/service/TimetableRepositoryIndex.java

```diff
@@ -161,6 +161,16 @@ Trip getTripForId(FeedScopedId tripId) {
     return tripForId.get(tripId);
   }
 
+  /**
+   * Checks if the specified trip is contained within the index.
+   *
+   * @param tripId the identifier of the trip to check
```

Comment:
Prefer {@link FeedScopedId}

Reply:
Fixed in f40626194bfc67e2bfdac64f41eb2d69d881d397